en/news/_posts/2016-04-01-ruby-2-1-10-released.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: news_post
+title: "Ruby 2.1.10 Released"
+author: "usa"
+translator:
+date: 2016-04-01 02:00:00 +0000
+lang: en
+---
+
+Ruby 2.1.10 has been released.
+This release is not for production environment, but for compatibility test with two-digit version number.
```

Comment:
This release is not intended for production use, but for compatibility tests with two-digit version numbers.